### ISSUE TITLE
version 45.0.7 missing chalk dependency

### ISSUE THREAD

**Issue by dtrodrigues** [nativefier/nativefier#1324]
**Homework**

- [ ] I looked at `nativefier --help` and https://github.com/nativefier/nativefier/blob/master/API.md
- [ ] I checked the [Troubleshooting section of the README](https://github.com/nativefier/nativefier/blob/master/README.md#troubleshooting).
- [x] I searched existing issues, open & closed. Yes, my bug is new.
- [x] I'm using the latest version available at https://github.com/nativefier/nativefier/releases

**Bug description**

**Steps to reproduce**

Give your ***full*** nativefier command and its logs, with the ***`--verbose` flag***, on a ***public*** site:

```
$ npm install -g nativefier@45.0.6

removed 3 packages, changed 227 packages, and audited 228 packages in 1s

37 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
$ nativefier --version
45.0.6
$ npm install -g nativefier@45.0.7

added 3 packages, changed 227 packages, and audited 231 packages in 851ms

37 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
$ nativefier --version
Error: Cannot find module 'chalk'
Require stack:
- /opt/homebrew/lib/node_modules/nativefier/lib/build/buildNativefierApp.js
- /opt/homebrew/lib/node_modules/nativefier/lib/main.js
- /opt/homebrew/lib/node_modules/nativefier/lib/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/opt/homebrew/lib/node_modules/nativefier/src/build/buildNativefierApp.ts:4:1)
    at Module._compile (node:internal/modules/cjs/loader:1097:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1149:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
```

**Expected behavior**

Nativefier to run

**Actual behavior**

Stacktrace shown above

**Debug info**

- Console logs of your `nativefier` build command, with `--verbose` flag
- If the bug happens at app run time, the in-app DevTools console logs (open it with F12)
- Error messages
- Screenshots
- Anything else relevant!

**Context**

 - Nativefier: 45.0.7
 - Node.js: (for example: 14.6.0)
 - Npm: 8.1.4
 - OS: macOS 11.6
 - Is it a regression? If yes, what's the last working / first broken version? Yes -- this did not occur on 45.0.6. 
 - Additional context: (for example: "I'm behind a proxy, with configuration X and protocol Y")

**Comment by ronjouch** [comment]
Thanks! Blergh, sorry 😄. Fixed, will ship in 45.0.8 in npm in a few minutes.

**Comment by ronjouch** [comment]
@dtrodrigues ✔, [45.0.8 is published](https://github.com/nativefier/nativefier/runs/4439727369). Maybe not yet visible in npm because of npm's aggressive cache, but will soon be. Thanks again for the ffffast report.
